### PR TITLE
Linking to HackerOne bug bounty

### PR DESCRIPTION
**Pull request by PROTechThor**
Addressing these issues:
https://trac.torproject.org/projects/tor/ticket/30601
https://dip.torproject.org/torproject/web/tpo/issues/42

### CODE REVIEW

**Comment by PROTechThor** [comment]
@gusgustavo I've made requested edits :)

File: templates/contact.html [hunk 1]
```diff
@@ -77,7 +77,7 @@ <h4 class="text-white display-5">
       </div>
       <div class="row mt-3 mr-5">
         <h4 class="text-primary">{{ _('Report a security issue.') }}</h4>
-        <p class="text-tpo">{{ _("If you've found a security issue in one of our projects or in our infrastructure, please email tor-security@lists.torproject.org. If you want to encrypt your mail, you can get the GPG public key for the list by contacting tor-security-sendkey@lists.torproject.org or from pool.sks-keyservers.net. Here is the fingerprint:") }}</p>
+        <p class="text-tpo">{{ _("If you've found a security issue in one of our projects or in our infrastructure, please email tor-security@lists.torproject.org. If you found a bug in Tor or Tor Browser, feel free to submit a bug report for our <a href="https://hackerone.com/torproject" target="_blank">{{ _('bug bounty program.') }}</a> If you want to encrypt your mail, you can get the GPG public key for the list by contacting tor-security-sendkey@lists.torproject.org or from pool.sks-keyservers.net. Here is the fingerprint:") }}</p>
```
Comment (gusgustavo) [inline templates/contact.html]: Sorry, we will need to be a little more specific here:"If you've found a security bug in Tor or Tor Browser, feel free to submit it for our..."

Comment (PROTechThor) [reply templates/contact.html]: @gusgustavo All done :)

File: templates/contact.html [hunk 2]
```diff
@@ -77,7 +77,7 @@ <h4 class="text-white display-5">
       </div>
       <div class="row mt-3 mr-5">
         <h4 class="text-primary">{{ _('Report a security issue.') }}</h4>
-        <p class="text-tpo">{{ _("If you've found a security issue in one of our projects or in our infrastructure, please email tor-security@lists.torproject.org. If you want to encrypt your mail, you can get the GPG public key for the list by contacting tor-security-sendkey@lists.torproject.org or from pool.sks-keyservers.net. Here is the fingerprint:") }}</p>
+        <p class="text-tpo">{{ _("If you've found a security issue in one of our projects or in our infrastructure, please email tor-security@lists.torproject.org. If you've found a security bug in Tor or Tor Browser, feel free to submit it for our <a href="https://hackerone.com/torproject" target="_blank">{{ _('bug bounty program.') }}</a> If you want to encrypt your mail, you can get the GPG public key for the list by contacting tor-security-sendkey@lists.torproject.org or from pool.sks-keyservers.net. Here is the fingerprint:") }}</p>
```
Comment (emmapeel2) [inline templates/contact.html]: There are some clashes between the different " and ' in this line, and the build breaks.

Comment (PROTechThor) [reply templates/contact.html]: @emmapeel2 Oof. Just noticed.

Comment (PROTechThor) [reply templates/contact.html]: @emmapeel2 Just made a new PR which I hope will solve this.

**Pull request merged**